### ISSUE TITLE
[Internationalization] Some languages use the incorrect specifiers for replacing data (%d instead of _START_, etc)

### ISSUE THREAD

**Issue by afestupinanp** [DataTables/Plugins#562]
I found out that some of the translations use the %d formats, instead of the _START_, _END_ or other special elements that indicate number of records, total of records, etc. This is especially notable on the spanish translations.

This is how it looks using the default english localization. Notice it shows the correct amount of records in the datatable.

![image](https://user-images.githubusercontent.com/89429713/230800576-3a9a06c5-ad7b-46b9-8180-3fef82d9c9bc.png)

Now, if we load the spanish localization provided by DataTables (es-CO).

![image](https://user-images.githubusercontent.com/89429713/230800611-04348efe-4daf-4dbe-8386-e115ddf3d0ac.png)

This is not the case for all the languages. For example, the German localization (de-DE)

![image](https://user-images.githubusercontent.com/89429713/230800668-46de9002-b8b4-46e3-831f-f1a3b263a7af.png)

**Comment by afestupinanp** [comment]
Seems to be related to #485, which is open since 2020 lol.

**Comment by AllanJard** [comment]
The German one looks correct to me: https://datatables.net/plug-ins/i18n/German.html . I'm not sure what I'm missing?

**Comment by afestupinanp** [comment]
Indeed, what I'm referring to is that some of the languages, such as es-CO, use the %d specifiers.

**Comment by AllanJard** [comment]
Thanks! And apologies I got confused.

I've corrected that one now. Looking through our database I think that's the only one? If you see some fixes that I haven't, then you can contribute changes [here](https://datatables.net/plug-ins/i18n/). I obviously don't know all of those languages, so I rely upon the community to provide contributions.
